lua/wire/server/wirelib.lua

```diff
@@ -1216,9 +1216,15 @@ function WireLib.AddOutputAlias( class, old, new )
 	ENT_table.OutputAliases[old] = new
 end
 
+local function effectiveMass(ent)
+	if not isentity(ent) then return 1 end
+	if ent:IsWorld() then return 99999 end
+	if not IsValid(ent) or not IsValid(ent:GetPhysicsObject()) then return 1 end
```

Comment:
Late. Anyways, I suggest to store the value `ent:GetPhysicsObject()` into own variable and separate if statement, it would make it both safer from race conditions and faster (because for example, `ent:GetPhysicsObject()` call on the line below might become invalid in very rare situations).